### ISSUE TITLE
Provide more details for `CONNECTION_<ID>_AUTH_<AUTHID>_(SCHEME|ID)` in docs

### ISSUE THREAD

**Issue by The-Judge** [elkozmon/zoonavigator#75]
Hey everyone,

after investing quite a long time now, I finally found a hint on how to use CONNECTION_<ID>_AUTH_<AUTHID>_SCHEME and CONNECTION_<ID>_AUTH_<AUTHID>_ID options. The existing documentation is not really informative.

In #60 , it seems as if at least a few hints are provided on how to use these parameters, but I think:

1. These info are missing in the documentation.
2. A full list of valid parameters would be nice.

**Comment by elkozmon** [comment]
I agree, the documentation could be improved a lot. Eventually, I will gladly do that. However in the meantime, I'm also happy to accept PRs in case anyone's interested - docs are written in reStructuredText and can be found in [docs](https://github.com/elkozmon/zoonavigator/tree/master/docs) folder.

**Comment by NilutpalBoro** [comment]
I am unable to login to remote zookeeper server since i have set the following in zookeeper server side  -Dzookeeper.allowSaslFailedClients=false -Dzookeeper.sessionRequireClientSASLAuth=true

I have set the java.security.auth.login.config to jaas config file 
Client {
       org.apache.zookeeper.server.auth.DigestLoginModule required
       username="testUser"
       password="pass123";
};

When i try to login i cant and i see the following error on zookeper logs
 ERROR [NIOWorkerThread-2:o.a.z.s.AuthenticationHelper@121] - Client authentication scheme(s) [ip] does not match with any of the expected authentication scheme [sasl], closing sessio
 
So I am trying to set the preset auth as 
snap set zoonavigator zoonavigator.connection.aclzk.auth.aclzkauth.scheme=sasl
error: cannot perform the following tasks:
- Run configure hook of "zoonavigator" snap (run hook "configure":
-----
Invalid Auth 'aclzkauth' configuration in Connection 'aclzk'
Make sure following configuration options are set: zoonavigator.connection.aclzk.auth.aclzkauth.scheme zoonavigator.connection.aclzk.auth.aclzkauth.id
-----)

where am I going wrong and how do i set the preset auth info
